app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -519,8 +428,6 @@ class EditPatientScreen : BaseScreen<
     identifiers.forEach { identifier -> inflateBpPassportView(identifier) }
 
     bpPassportsLabel.visibleOrGone(identifiers.isNotEmpty())
-
-    post { requestLayout() }
```

Comment:
Would it be possible to check why this change was added and how it's safe to remove now?

Reply:
Not entirely sure why it was added before, but when we are inflating and adding child views it will automatically request layout internally. So there is no need to manually call it anymore.

I have tested the BP passports layout (single & multiple) and it is working fine.